autosubmit/platforms/platform.py

```diff
@@ -126,25 +125,15 @@ def __init__(self, expid, name, config, auth_password=None):
         self.get_cmd = None
         self.put_cmd = None
         self._submit_hold_cmd = None
-        self._submit_command_name = None
         self._submit_cmd = None
         self._submit_cmd_x11 = None
         self._checkhost_cmd = None
         self.cancel_cmd = None
-        self.otp_timeout = None
         self.two_factor_auth = None
-        self.otp_timeout = self.config.get("PLATFORMS", {}).get(self.name.upper(), {}).get("2FA_TIMEOUT", 60 * 5)
```

Comment:
These properties removed above all stay, I believe. They are declared here in the parent, to be used by implementations like Paramiko and grand-children classes (slurm, pbs, etc.).